project/CakeDependencies.scala

```diff
@@ -0,0 +1,50 @@
+// Copyright: 2017 https://github.com/cakesolutions/sbt-cake/graphs
+// License: http://www.apache.org/licenses/LICENSE-2.0
+
+import sbt._
+
+object CakeDependencies {
```

Comment:
For ease of searching, Lets place the top level val's and objects into alphabetical order